docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,72 @@
+=====================
+RPM rsync Distributor
+=====================
+
+Purpose:
+========
+The RPM rsync distributor publishes RPM content to a remote server. As the name suggests, the
+distributor uses rsync to copy.
+
+Configuration
+=============
+Here's example of rpm_rsync_distributor configuration:
+
+.. code-block:: json
+
+    {
+     "distributor_id": "rpm_rsync_distributor",
```

Comment:
For clarity this label could be set so that it's not the same value as `distributor_type_id`. Maybe this could be `my_rsync_distributor` or similar?